src/main/java/com/traveltime/sdk/dto/requests/timefilterfast/transportation/DrivingAndPublicTransport.java

```diff
@@ -0,0 +1,22 @@
+package com.traveltime.sdk.dto.requests.timefilterfast.transportation;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.traveltime.sdk.dto.requests.timefilterfast.Transportation;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+
+@Value
+@Builder
+@Jacksonized
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DrivingAndPublicTransport implements Transportation {
+    @Positive(message = "walkingTime must be greater than 0")
```

Comment:
I don't think we expose those parameters in any api yet.

Reply:
So I remove it everywhere, right?

Reply:
In `...timefilterfast.*`